static/css/syna.css

```diff
@@ -21,7 +23,8 @@
 
 #particles-js {
   position: absolute;
-  width: 90%;
+  margin-left: 0;
```

Comment:
As jumbotron is `padding-left: 2rem` within bootstrap we can work around the margin with `margin-left: -2rem`.
Would also love the take on this solution from @mpourismaiel

Reply:
Yeah Alright, will fix that.